contrib/platform/src/com/sun/jna/platform/win32/Variant.java

```diff
@@ -773,14 +773,15 @@ public ByReference() {
             }
 
             public ByReference(VARIANT[] variantArg) {
-                this.variantArg = variantArg;
+                super(variantArg);
             }
         }
 
-        public VARIANT[] variantArg = new VARIANT[1];
+        public VARIANT[] variantArg;
 
         public VariantArg() {
             super();
+            variantArg = new VARIANT[1];
```

Comment:
I am not sure about this, as the super call is now done after the field is initialized.

Reply:
https://stackoverflow.com/a/15682615

It seems like this is no issue at all, the members are allways initialzed after the super call, see link above.